tools/codespell/Dockerfile.builder

```diff
@@ -0,0 +1,8 @@
+# python
+FROM python:3.7-slim as python
```

Comment:
you will need to install `codespell` and `yamllint` in tools/Dockerfile.builder

Reply:
@arkodg 
Sorry for my miscomprehension, I thought the linters need to run as docker-in-docker(cause the tools/Dockerfile.builder only providers golang base image and has docker installed)

If we install codespell and yamllint in tools/Dockerfile.builder, does it mean we need a ubuntu base image instead of golang? (python is required for yamllint and codespell)
> FROM golang:1.18.2 as builder > FROM ubuntu:focal as builder

Reply:
Refer to Istio: https://github.com/istio/tools/blob/master/docker/build-tools/Dockerfile

I made some changes.